### ISSUE TITLE
finish styling

### ISSUE THREAD

**Issue by simonv3** [simonv3/glossy#8]

**Comment by RichardLitt** [comment]
Going to need better concepts instead of books for the images.

**Comment by RichardLitt** [comment]
Going to need to find a really good / better header font.

**Comment by simonv3** [comment]
Yeah, I don't mind the book one, but the books one will have to be replaced, one of them at least. And probably sourced correctly. 

You mean the logo? I like the logo.
